### PR TITLE
[Fix and Suggestion]Rework WorldGen drop

### PR DESCRIPTION
**Pull request by AlasDiablo**
Hi, i'm currently working on a mod who add compatibility with Just Enough Resources and i have encountered some problem.

In this pr i have fix some issues and make some proposition of amelioration on drop display.

Have a nice day, and i will hope this can help you !
Sorry for my bad english.

Change with vanila ore:
Iron Ore

Change and fix with compatibility by mod creator ore:
Certus Quartz

### CODE REVIEW

**Comment by Seniorendi** [comment]
This is really neat

File: src/api/java/jeresources/api/drop/LootDrop.java [hunk 1]
```diff
@@ -186,12 +186,21 @@ private String getDropChance() {
...
         if (chance < 10) return String.format("%.1f", chance);
         return String.format("%2d", (int) chance);
     }
 
+    /**
+     * Use by world gen only
+     * Check if conditionals contains Conditional.affectedByFortune
+     * @return True if Conditional.affectedByFortune is found, False if it not
+     */
+    public boolean isAffectedByFortune() {
```
Comment (way2muchnoise) [inline src/api/java/jeresources/api/drop/LootDrop.java:200]: Can we also add a general function for this, seems like a useful tool to me.
`public boolean isAffectedBy(Conditional)`

**Review by way2muchnoise — CHANGES_REQUESTED** [review]
Looks good to me with just the small change mentioned for generalisation.

**Review by way2muchnoise — CHANGES_REQUESTED** [review]
One more change, could you bump the revision number in the following file https://github.com/way2muchnoise/JustEnoughResources/blob/master/gradle.properties#L5

**Review by way2muchnoise — APPROVED** [review]

**Comment by way2muchnoise** [comment]
Thanks for the PR, new build with this new UI will be up on CF soon :)

**Comment by Seniorendi** [comment]
Nice!

**Pull request merged**